stone/backends/js_helpers.py

```diff
@@ -49,11 +49,11 @@ def fmt_obj(o):
         return json.dumps(o, indent=2)
 
 
-def fmt_error_type(data_type):
+def fmt_error_type(data_type, wrap_error_in=''):
```

Comment:
nit: Can't we just default this to `wrap_error_in='Error'`

Reply:
We do need to default to '' becuase that is what comes in if the flag isn't set, so this would have to be checked when we call it and I think its better to abstract that here